asset_manager/storage.py

```diff
@@ -44,26 +42,26 @@ def _bucket_from_argument(
 
 
 def write_string_to_object(
-    object_name: str,
-    text: Union[str, bytes],
-    bucket: Optional[BucketLikeType] = None
+    object_name: str, text: Union[str, bytes], bucket: Optional[BucketLikeType] = None
 ) -> None:
-    '''
+    """
     Write a string into an object in S3.
-    '''
+    """
     bucket = _bucket_from_argument(bucket)
     if isinstance(text, str):
         text = text.encode()
-    bucket = _s3().Bucket(conf['S3_BUCKET'])
+    bucket = _s3().Bucket(conf["S3_BUCKET"])
     _default_bucket()
-    bucket.put_object(Key=object_name, Body=text)
+    try:
+        bucket.put_object(Key=object_name, Body=text)
+    except NoCredentialsError as exc:
+        raise Exception("No s3 credentials, you may need to sign in") from exc
```

Comment:
I ran into this `NoCredentialsError` when I hadn't set up my new computer for AWS access yet; this improved error should make the next steps a little more obvious next time this happens.